.gitattributes

```diff
@@ -1,8 +1,10 @@
+* text=auto eol=lf
```

Comment:
We actually want Git to change the line-endings for Toit files, so that we have test-coverage for Windows line-endings on the build bots.
We won't be able to prevent users to use non-lf line endings on Windows, so we need to test it too.